Documentation/ApiOverview/Localization/TranslationServer/Crowdin/OnlineTranslation.rst

```diff
@@ -1,140 +1,194 @@
+    consistency. Make extensive use of the terms and glossary.
 
-5. Be consistent. Localization of high quality is characterised by the consistency. Make extensive use of the terms and glossary.
+#.  Use machine translation carefully. It is tempting but dangerous to do
+    a quick translation with one of the common machine translation tools and
+    sometimes it can help you to get a deeper understanding of the meaning of a
+    text. But very often a machine-translated text breaks all the above rules
+    unless you rework the text carefully afterwards.
 
-6. Make careful use of machine translation. It’s tempting but dangerous to do a quick translation with one of the common machine translation tools and it can sometimes help you getting a deeper understanding of the meaning of a text.
-   But very often a machine translated text breaks all of the rules above, unless you carefully rework the text afterwards.
+#.  Work together. As in all other aspects of Open Source, things get so much
+    better when we work together. So, reach out for help when you get stuck. Or
+    offer your knowledge if someone ask for it. Crowdin provides a good platform
+    for collaborating with your team translators, and please join the
+    `Translation Slack channel #typo3-translations`_.
 
-7. Work together. As in all other aspects of Open Source, things get so much better, if we work together. So, reach out for help if you get stuck. Or offer your knowledge if someone ask.
-   Crowdin offers a good platform for cooperating with your team translators, and please join the `Translation Slack channel <https://typo3.slack.com/archives/C032FRT0W>`__.
 
 Translation styles
 ==================
 
-In general and where it makes sense, we follow the style guide from the Content Team: `<https://typo3.org/community/teams/content/writing-style-guide/>`__
+In general, and where it makes sense, we follow the
+:ref:`Writing Style Guide <t3content:writing-style-guide>` from the Content Team.
+
+In the future (when translation teams start getting bigger), it might be a good
+idea to develop local style guides.
 
-In the future (when translation teams start getting bigger), it might be a good idea to develop local style guides.
 
 Links
 =====
 
-There are many good resources when it comes to translation, language, dictionaries, etc. Feel free to suggest your favorite websites, when you work with language.
-
-`<https://iate.europa.eu/home>`__
-
-`<https://gengo.com/translators/resources/>`__
+There are many good resources when it comes to translation, language,
+dictionaries, etc. Feel free to suggest your favorite websites, when you work
+with language.
 
-`<https://docs.microsoft.com/style-guide>`__
+*   `<https://iate.europa.eu/home>`__
+*   `<https://gengo.com/translators/resources/>`__
+*   `<https://docs.microsoft.com/style-guide>`__
 
 
 FAQ
 ===
 
 
-Should I localize both 11.5 and main?
+Should I localize both 12.4 and main?
```

Comment:
Should we backport this to 11.5?

Reply:
Yes, have it on the radar to adjust it before merge.